SYCL/ESIMD/dpas/dpas_tf32.cpp

```diff
@@ -0,0 +1,87 @@
+//==---------------- dpas_tf32.cpp  - DPC++ ESIMD on-device test ----------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu-intel-pvc || esimd_emulator
+// UNSUPPORTED: cuda || hip
+// RUN: %clangxx -fsycl -DESIMD_XE_HPC %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// XFAIL: esimd_emulator
+
+// The test verifies the low-level API for DPAS with 'tfloat32' types.
+// It checks the versions of DPAS with and without the accumulator operand.
+
+#include <sycl/ext/intel/esimd.hpp>
+#include <sycl/sycl.hpp>
+
+using namespace sycl;
+using namespace sycl::ext::intel::esimd;
+using namespace sycl::ext::intel::experimental::esimd;
+
+int main() {
+  queue Q;
```

Comment:
why not use custom selector and exception handler as in ESIMD tests?
` queue q(esimd_test::ESIMDSelector{}, esimd_test::createExceptionHandler());`

Reply:
because the way how CI works, the GPU selector will be chosen anyway.
Regarding exception handler..., Ok, I'll add it.

Reply:
Fixed in 21db007d31884e169792e7c0d992e324202bb010